BasicGates/Tests.qs

```diff
@@ -119,22 +125,23 @@ namespace Quantum.Kata.BasicGates {
             AssertAllZero(qs);
         }
     }
+
     
-    
+
     // ------------------------------------------------------
-    operation T17_BellStateChange1_Test () : Unit {
+    operation T18_BellStateChange1_Test () : Unit {
         VerifyBellStateConversion(BellStateChange1, 0, 1);
     }
     
     
     // ------------------------------------------------------
-    operation T18_BellStateChange2_Test () : Unit {
+    operation T19_BellStateChange2_Test () : Unit {
```

Comment:
The `T...` prefix was introduced to order the tests when they show up in Visual Studio Test panel (or in VS Code). Initially there were less than 10 tasks in the first part, so the prefix was T - part number - 1-digit task number. The task you're adding is the 10th one, so the `T110_` task will appear immediately after `T11_`, which is not as neat as it could be.

Could you please go over the test operations in this file (and in the matching `%kata` entries in the notebook) and rename them to have 2-digit task numbers? For example,
```suggestion
    operation T109_BellStateChange2_Test () : Unit {
```
Thank you!